dev/io.openliberty.org.jboss.resteasy.common/src/org/jboss/resteasy/specimpl/ResteasyHttpHeaders.java

```diff
@@ -24,7 +24,42 @@
+       mtCache.put(MediaType.SERVER_SENT_EVENTS_TYPE.toString(), Collections.singletonList(MediaType.SERVER_SENT_EVENTS_TYPE));                   // Liberty change
+       mtCache.put(MediaType.TEXT_HTML_TYPE.toString(), Collections.singletonList(MediaType.TEXT_HTML_TYPE));                                     // Liberty change
+       mtCache.put(MediaType.TEXT_PLAIN_TYPE.toString(), Collections.singletonList(MediaType.TEXT_PLAIN_TYPE));                                   // Liberty change
+       mtCache.put(MediaType.TEXT_XML_TYPE.toString(), Collections.singletonList(MediaType.TEXT_XML_TYPE));                                       // Liberty change
+       mtCache.put(MediaType.WILDCARD_TYPE.toString(), Collections.singletonList(MediaType.WILDCARD_TYPE));                                       // Liberty change
+       mediaTypeCache = Collections.unmodifiableMap(mtCache);                                                                                     // Liberty change
+                                                                                                                                                  // Liberty change
+       Map<String,List<Locale>> langCache = new HashMap<>();                                                                                      // Liberty change
+       langCache.put(Locale.CHINESE.toString(), Collections.singletonList(Locale.CHINESE));                                                       // Liberty change
+       langCache.put(Locale.ENGLISH.toString(), Collections.singletonList(Locale.ENGLISH));                                                       // Liberty change
+       langCache.put(Locale.FRENCH.toString(), Collections.singletonList(Locale.FRENCH));                                                         // Liberty change
+       langCache.put(Locale.GERMAN.toString(), Collections.singletonList(Locale.GERMAN));                                                         // Liberty change
+       langCache.put(Locale.ITALIAN.toString(), Collections.singletonList(Locale.ITALIAN));                                                       // Liberty change
+       langCache.put(Locale.JAPANESE.toString(), Collections.singletonList(Locale.JAPANESE));                                                     // Liberty change
+       langCache.put(Locale.KOREAN.toString(), Collections.singletonList(Locale.KOREAN));                                                         // Liberty change
+       langCache.put(Locale.SIMPLIFIED_CHINESE.toString(), Collections.singletonList(Locale.SIMPLIFIED_CHINESE));                                 // Liberty change
+       langCache.put(Locale.TRADITIONAL_CHINESE.toString(), Collections.singletonList(Locale.TRADITIONAL_CHINESE));                               // Liberty change
+       languageCache = Collections.unmodifiableMap(langCache);                                                                                    // Liberty change
```

Comment:
Should `langCache.put("", LANGUAGE_WILDCARD);` be added?

Reply:
yup, done